cmd/loop/reservations.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+
+	"github.com/lightninglabs/loop/looprpc"
+	"github.com/urfave/cli"
+)
+
+var reservationsCommands = cli.Command{
+
+	Name:      "reservations",
+	ShortName: "r",
+	Usage:     "manage reservations",
```

Comment:
nit: A bit more help could be useful so users understand what the reservations are etc.